src/components/custom-tree-item.jsx

```diff
@@ -96,6 +135,16 @@ CustomContent.propTypes = {
      * The callback to call when handle Selection Click.
      */
     onSelect: PropTypes.func,
+
+    /**
+     * The callback to call when handle Context Menu Click.
+     */
+    onContextMenu: PropTypes.func,
```

Comment:
onAddIconClick instead of onContextMenu, it's an implementation detail that this button open the same menu as the contextual one.

Reply:
I changed onContextMenu. I don't think it's a good idea to use another DirectoryTreeContextualMenu ( a lot of code to be duplicated)